runtime/src/drivers.rs

```diff
@@ -66,6 +67,8 @@ pub struct Drivers {
 
     pub pcr_bank: PcrBank,
 
+    pub pcr_reset: PcrResetCounter,
```

Comment:
These should persist across Caliptra warm/update resets. I think they should be added to the `PersistentData` driver (as runtime-only, see DPE state for an example)